### ISSUE TITLE
Issue in language and date labelling with ontonotes NER model

### ISSUE THREAD

**Issue by shv07** [flairNLP/flair#1811]
enhancement.

I have been using the pretrained 'ner-ontonotes' NER model and I noticed these unexpected results.

1. Date - dates in format dd-mm-yyy are not recognised as 'DATE' by the model however the format May 7, 2020 works fine.

2. Languages - a sentence with multiple languages gives unexpected tags (I got 'PERSON') with same pre-trained NER model. 
example - "I know English, Hindi, Bengali" - the model is not able to identify all the languages. However, if there's a space before each punctuation, the predictions are correct (i.e. when the sentence is - I know English , Hindi , Bengali )

**Comment by alanakbik** [comment]
Are you using the tokenizer when predicting? Up until flair 0.5.1 you must explicitly state `use_tokenizer=True` to use it: 

```python
from flair.data import Sentence

# Make a sentence object by passing an untokenized string and the 'use_tokenizer' flag
sentence = Sentence('The grass is green.', use_tokenizer=True)

# Print the object to see what's in there
print(sentence)
```

With the upcoming 0.6 release of Flair, tokenization is turned on by default, so you no longer have to explicitly set `use_tokenizer`.

**Comment by shv07** [comment]
Thanks for the quick response.

Using `use_tokenizer=True` solves the issue in language labelling but with dates it's more or less the same. It might be because of the training data I guess.

For example - 
```
sentence = Sentence('date: 15-08-1998', use_tokenizer=True)

tagger = SequenceTagger.load('ner-ontonotes')

tagger.predict(sentence)
```
It assigns the label 'CARDINAL' to token 15-08-1998.

**Comment by alanakbik** [comment]
Yes the first issue is likely because of the training data. Not sure if this date format appears in Ontonotes.

**Comment by shv07** [comment]
Yeah. Thanks for the help.
